src/objects/control.ts

```diff
@@ -103,9 +103,14 @@ export default class Control {
     if (this.descs) {
       Object.entries(this.descs).forEach(([key, desc]) => {
         if (desc) {
-          result += `  desc "${key}", "${escapeDoubleQuotes(
-            removeNewlinePlaceholders(desc)
-          )}"\n`;
+          if(key.match("default") && this.desc) {
+            console.error(`${this.id} has a redundant default value. This command will write the "desc" value to the control and will not write the "default" value.`);
```

Comment:
This is what is expected. The new standard is to use the .descriptions.... object , the main desc object is there for backwards compatibility.

Reply:
you can review the profile schema or the ruby inspec-objects as a reference if needed in the future.